packages/components/src/components/DiagramSequence.vue

```diff
@@ -185,6 +185,7 @@ export default {
       expandedActions.delete(undefined);
 
       for (let index = 0; index < result.actions.length; index++)
+        // eslint-disable-next-line vue/no-side-effects-in-computed-properties
         this.$set(this.collapsedActions, index, !shouldExpand(result.actions[index]));
```

Comment:
This is pretty worrisome. References to other properties within a computed field causes this method to be re-run any time the referenced data is changed (`collapsedActions`). This is likely introducing some unintended recursion. At least it seems to be bounded.

Reply:
If this is producing a warning I think we should let it remain

Reply:
My change was to silence the warning because I didn't want to spend too much time to figure out the correct fix. Do you think we should?